### PR TITLE
Support limiting request body size in registry mgmt API

### LINKED ISSUE #2718: Add max payload size to all device registry requests
Currently there is no payload limit for requests to the device registry. Therefore it would be possible to upload and store multible megabytes worth of data, inside a device json for example.
To prevent that, a config property could be added which defines the maximum payload size for device registry requests.
This could work in a similar way to the `maxPayloadSize` config property already present for up/downstream messages.

### PR DESCRIPTION
**Pull request by sophokles73**
The device registry implementations have been extended to allow limiting
the body size of incoming HTTP requests to the Device Registry
Management API.

Fixes #2718

### CODE REVIEW

File: tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java [hunk 1]
```diff
@@ -126,6 +127,24 @@ public void testAddTenantSucceedsWithAGeneratedId(final VertxTestContext context
...
     }
 
+    /**
+     * Verifies that a request to create a tenant with a body that exceeds the registry's max payload limit
+     * fails with a {@link HttpURLConnection#HTTP_ENTITY_TOO_LARGE} status code.
+     *
+     * @param context The vert.x test context.
+     */
+    @Test
+    public void testAddTenantFailsForRequestPayloadExceedingLimit(final VertxTestContext context)  {
```
Comment (kaniyan) [inline tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: How about a test also to check for limit during _update Tenant_ operation?

Comment (sophokles73) [reply tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: sure, I'll add one

Comment (sophokles73) [reply tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: I have pushed a change ...

Comment (sophokles73) [reply tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: @kaniyan Would you mind taking another look?

Comment (kaniyan) [reply tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: @sophokles73 There are file conflicts. Can you please rebase?

Comment (sophokles73) [reply tests/src/test/java/org/eclipse/hono/tests/registry/TenantManagementIT.java:137]: done

**Review by kaniyan — APPROVED** [review]
LGTM

**Pull request merged**